R/interface.R

```diff
@@ -2,12 +2,19 @@
 
 # PCC matrix c wrapper
 PCC <- function(aM, bM = NULL, use = NULL, asMatrix = TRUE, debugOn = FALSE) {
-  if(is.null(bM)) bM <- aM
+  n = as.integer(nrow(aM))
+  m = as.integer(ncol(aM))
+  if(is.null(bM)) {
+    p = 0
+    bM <- aM
+  } else {
+    p = as.integer(ncol(bM))
+  }
   res <- .C("R_pcc_matrix", aM = as.double(aM),
                             bM = as.double(bM),
-                            n = as.integer(nrow(aM)), # nInd
-                            m = as.integer(ncol(aM)), # nPhe A
-                            p = as.integer(ncol(bM)), # nPhe B
+                            n, # nInd
```

Comment:
These 3 lines have been unnecessarily changed, why touch lines of code when nothing happens ??? 
Also all as.XX calls should happen in the .C interface, since in R types like double and integer don't exist (and storage classes might be dynamically changed so the statement p = 0, is not gauranteed to be an integer now)